### ISSUE TITLE
Store artifacts

### ISSUE THREAD

**Issue by sgurlt** [appleboy/ssh-action#38]
Hey,

is there a way to create artifacts while connected via ssh?

**Comment by jbnjohnathan** [comment]
I would like this as well, or just some way to get the output of the SSH commands back to the container running the github action

**Comment by appleboy** [comment]
Can you describe any scenario?

**Comment by TheGrandWazoo** [comment]
Looking for the same thing as well...

Like your example:
```
script: |
  whoami
```
Is it possible to do:
```
-name: Who
  id: who
  script: |
    echo "::set-output name=id::$(whoami)"
-name: Print
  run: echo ${{ steps.who.outputs.id }}
```
Currently trying it and it does run the command but no output id provided on the print.

**Comment by jbnjohnathan** [comment]
Yeah for me I would also like to capture output from the SSH session to use as input to other scripts I will run in the github action container.
The solution I had to do now was to run SSH "manually" like this
`echo ::set-env name=MYENV::$(ssh -i mykey.key -o StrictHostKeyChecking=no user@hostname "whoami")`

**Comment by haidertm** [comment]
Was any solution found for this issue/scenario, where we can share info or file from server to artifacts of github?

**Comment by BoberMod** [comment]
I need to store output from the SSH session and post it into GitHub PR comments later.
Can I archive that with this GH Action?
